themes/theme-next/src/node/config/templateBuildRenderer.ts

```diff
@@ -0,0 +1,37 @@
+import type { TemplateRendererContext } from 'vuepress/utils'
```

Comment:
This file could be extracted to @vuepress/theme-helper

Reply:
I think `@vuepress/theme-helper` can be included in future update plans.

Reply:
A lot of functions can be extracted to `@vuepress/theme-helper`, I think we can start this migration in this pr, and cherry pick the changed to main continuously